test/integration/ComputeFlow.test.ts

```diff
@@ -267,7 +285,7 @@ describe('Compute flow', () => {
     assert(ddo.dataToken === tokenAddress, 'ddo.dataToken !== tokenAddress')
     const storeTx = await ocean.onChainMetadata.publish(ddo.id, ddo, alice.getId())
     assert(storeTx)
-    await sleep(aquaSleep)
+    await waitForAqua(ocean, ddo.id)
```

Comment:
Why we need to wait for aqua after the assert ? specially that nothing won't happen after waiting for aqua.

Reply:
after publishing an asset, you are going to use it somehow.  But aqua is scanning the chain every 15-20 secs, so first we need to wait, to make sure that aqua has cached the asset